include/sdf/ParserConfig.hh

```diff
@@ -188,13 +192,21 @@ class SDFORMAT_VISIBLE ParserConfig
   /// If the FindFileCallback provides a non-empty string, the URI will be
   /// stored in the DOM object, and the original (unresolved) URI will be
   /// stored in the underlying Element.
-  public: void SetStoreResovledURIs(bool _resolveURI);
+  public: void SetStoreResolvedURIs(bool _resolveURI);
 
   /// \brief Get the storeResolvedURIs flag value.
   /// \return True if the parser will attempt to resolve any URIs found and
   /// store them.  False to preserve original URIs
   public: bool StoreResolvedURIs() const;
 
+  /// \brief Set the path to the SDF file.
+  /// \param[in] _path Full path to SDF file.
+  public: void SetFilePath(const std::string &_path);
+
+  /// \brief Get the path to the SDF file.
+  /// \return Full path to SDF file.
+  public: const std::string &FilePath() const;
```

Comment:
I would be more specific with this naming, like `URIRelativePath` because `FilePath` is already used in the `Element` class and many other DOM classes that are loaded directly from a file, but since `ParserConfig` represents configuration, not data loaded from a file, I would use a different name

Reply:
I gave a shot at naming: `SetRelativeURISearchPath`. Not tied to it, happy to change to a better name.

Reply:
looks good